pkgs/applications/networking/iptraf-ng/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.4";
+  name = "iptraf-ng-${version}";
+
+  src = fetchurl {
+    url = "https://fedorahosted.org/releases/i/p/iptraf-ng/${name}.tar.gz";
+    sha256 = "02gb8z9h2s6s1ybyikywz7jgb1mafdx88hijfasv3khcgkq0q53r";
+  };
+
+  buildInputs = [ ncurses ];
+
+  configurePhase = ''
```

Comment:
I'll admit that I packaged it last night while I was dead tired. If you don't like it then let me know what I should change and I'll amend it accordingly. :)

On 2 October 2015 10:39:57 BST, lethalman <notifications@github.com> wrote:
>> @@ -0,0 +1,44 @@
>> +{ stdenv, fetchurl, ncurses }:
>> +
>> +stdenv.mkDerivation rec {
>> +  version = "1.1.4";
>> +  name = "iptraf-ng-${version}";
>> +
>> +  src = fetchurl {
>> +    url =
>"https://fedorahosted.org/releases/i/p/iptraf-ng/${name}.tar.gz";
>> +    sha256 = "02gb8z9h2s6s1ybyikywz7jgb1mafdx88hijfasv3khcgkq0q53r";
>> +  };
>> +
>> +  buildInputs = [ ncurses ];
>> +
>> +  configurePhase = ''
>
>Ok so it's a kind of customized configure I guess, since prefix is
>`$out` by default.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/NixOS/nixpkgs/pull/10173/files#r41005570